### PR TITLE
Release Chef for MVP resources

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
Release Chef for MVP resources

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass (`$ rake test`).
  - [ ] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [ ] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

**Review by AsisBagga — APPROVED** [review]
LGTM

**Comment by nabhajit-ray** [comment]
> ### Description
> Release Chef for MVP resources
> 
> ### Issues Resolved
> ### Check List
> * [ ]  New functionality includes testing.
>   
>   * [ ]  All tests pass (`$ rake test`).
>   * [ ]  New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
> * [ ]  New functionality has been documented in the README if applicable.
>   
>   * [ ]  New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
> * [ ]  Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

**Review by nabhajit-ray — APPROVED** [review]

**Review by VenkateshRavula — CHANGES_REQUESTED** [review]
Please arrange imagestreamer resource together in changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,8 @@
-## 3.4.0(unreleased)
+## 3.4.0
 ### Notes
 - This release extends support of the SDK to OneView REST API version 1600 (OneView v5.20) and (ImageStreamer v5.20).
```
Comment (VenkateshRavula) [inline CHANGELOG.md]: remove this " and (ImageStreamer v5.20)", since we already mentioned it below

**Review by chebroluharika — CHANGES_REQUESTED** [review]
Please change description in README for image streamer resources.
Change it to - For image_streamer resources: HPE Synergy Image Streamer appliance (API version 300, 500, 600, 800, 1020, 1600)

**Comment by VenkateshRavula** [comment]
> Please change description in README for image streamer resources.
> Change it to - For image_streamer resources: HPE Synergy Image Streamer appliance (API version 300, 500, 600, 800, 1020, 1600)

also add 1000 to the list
Done, please check

File: CHANGELOG.md (hunk 1 above)
Comment (yuvirani) [reply CHANGELOG.md]: Updated.

**Review by chebroluharika — APPROVED** [review]

**Review by VenkateshRavula — APPROVED** [review]

**Pull request merged**